src/condor_ce_jobmetrics

```diff
@@ -78,6 +93,28 @@ def get_ads(spooldir):
     return [ads, environ]
 
 
+def dict_sum1(src, dst):
+    for key, val in src.items():
+        dst[key] += val
+
+
+def dict_sum2(src, dst):
```

Comment:
This function's a little opaque, could you add a docstring that explains its purpose?

Reply:
> This function's a little opaque

_\[unlike the code blocks it came from\]_

> could you add a docstring that explains its purpose?

you got it boss!  :)